content/en/docs/Writing policies/match-exclude.md

```diff
@@ -120,6 +120,19 @@ spec:
 ```
 
 As of Kyverno 1.5.0, wildcards are supported in the `kinds` field allowing you to match on every resource type in the cluster.
+Also, Selector labels supported Wildcard `(* or ?)` for key as well as value on following paths.
+* `pol.spec.match.resources.selector.matchLabels`
```

Comment:
Remove the `pol.spec.` portion, please. Just start with either `match` or `exclude`.

Reply:
yup